unified-installer/astra-unified-installer.sh

```diff
@@ -2318,6 +2345,15 @@ step_generate_astra_connector_yaml() {
     if [ -n "$_PROCESSED_LABELS" ]; then
         labels_field_and_content="${__NEWLINE}  labels:${__NEWLINE}${_PROCESSED_LABELS}"
     fi
+    local cluster_line=""
+    if [ -n "$cluster_id" ]; then
+        cluster_line="clusterId: $cluster_id"
+    elif [ -n "$cluster_name" ]; then
+        cluster_line="clusterName: $cluster_name"
+    else
+        fatal "no clusterId or clusterName found"
+    fi
```

Comment:
Since the user can provide either a cluster ID or cluster name, we need to change which one is set in the connector CR depending on which is provided.